r/tests/testthat/test-dplyr-group-by.R

```diff
@@ -28,9 +28,9 @@ test_that("group_by groupings are recorded", {
       group_by(chr) %>%
       select(int, chr) %>%
       filter(int > 5) %>%
+      collect() %>%
       summarize(min_int = min(int)),
```

Comment:
Sorry @ianmcook , didn't spot this one the first time round

```suggestion
      collect(),
```